src/github.com/getlantern/balancer/dialer.go

```diff
@@ -60,7 +56,7 @@ type dialer struct {
 func (d *dialer) Start() {
 	d.consecSuccesses = 1 // be optimistic
 	d.closeCh = make(chan struct{})
-	d.checkTimer = time.NewTimer(maxCheckTimeout)
+	d.checkTimer = time.NewTimer(MaxCheckTimeout)
```

Comment:
While we're in here, it would be nice to randomize this like we do with the config fetcher.